src/main/java/bio/terra/workspace/service/job/StairwayExceptionSerializer.java

```diff
@@ -11,10 +11,13 @@
 import java.lang.reflect.InvocationTargetException;
 import java.util.List;
 import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.http.HttpStatus;
 
 public class StairwayExceptionSerializer implements ExceptionSerializer {
-  private ObjectMapper objectMapper;
+  private static final Logger logger = LoggerFactory.getLogger(StairwayExceptionSerializer.class);
```

Comment:
Do we need this logger if we aren't logging anything yet?

Reply:
No, I did some logging for debugging. Removed the logging, but not the logger. I'll remove it.